### PR TITLE
Tasklist page includes TA specific sections and tasks

### PR DESCRIPTION
**Pull request by bryangaledxw**
# Changes in this PR

* This PR rearranges the sections pulled in from Approved Premises to fit Temporary Accommodation
* We also allow tasks to have distinct two forms of their name - a standard form used on the check your answers page and as heading captions, and a call-to-action form used on the task list page

### Before
localhost_3000_applications_89dc1fd7-07e5-4ea9-8b5e-cfbec383e744 (1)

### After
localhost_3000_applications_89dc1fd7-07e5-4ea9-8b5e-cfbec383e744

# Release checklist

[Release process documentation](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/edit-v2/4247847062?draftShareId=a1c360ab-bd31-4db1-aae3-7cc002761de9).

## Pre-merge checklist

- [ ] Are any changes required to dependent services for this change to work?
  (eg. CAS API)
- [ ] Have they been released to production already?

## Post-merge checklist

- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to test
- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to preprod
- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to prod

### CODE REVIEW

**Comment by bryangaledxw** [comment]
[Tasklist page includes TA specific sections and tasks](https://trello.com/c/oTN2UgfA/1097-tasklist-page-includes-ta-specific-sections-and-tasks)

File: cypress_shared/helpers/apply.ts [hunk 1]
```diff
@@ -88,12 +98,14 @@ export default class ApplyHelper {
   }
 
   completeApplication() {
+    this.completeAddSenetenceInformation()
```
Comment (tahb) [inline cypress_shared/helpers/apply.ts]: Typo on Sentence 👍

File: server/form-pages/apply/schema.json [hunk 2]
```diff
@@ -4,9 +4,32 @@
...
   "properties": {
+    "sentence-information": {
+      "type": "object",
+      "properties": {
+        "sentence-type": {
+          "type": "object",
+          "properties": {
+            "sentenceType": {
+              "enum": [
+                "bailPlacement",
```
Comment (tahb) [inline server/form-pages/apply/schema.json]: Figma only has 5 options here, I assume that difference doesn't matter too much this point as there are also other pages such as sentence length and sentence expiry.

File: cypress_shared/fixtures/applicationData.json [hunk 3]
```diff
@@ -4,6 +4,148 @@
...
+          "answer": "Some answer for the second RoSH question"
+        },
+        {
+          "questionNumber": "3",
+          "label": "The third RoSH question",
+          "answer": "Some answer for the third RoSH question"
+        }
+      ]
+    },
+    "offence-details": {
```
Comment (tahb) [inline cypress_shared/fixtures/applicationData.json]: This tab is called "Risk levels" in the prototype, I'm not sure if this difference is significant.

**Review by tahb — APPROVED** [review]
Looks good 👍

File: server/form-pages/apply/schema.json (hunk 2 above)
Comment (bryangaledxw) [reply server/form-pages/apply/schema.json]: Yep, definitely not taking these as done

File: cypress_shared/fixtures/applicationData.json (hunk 3 above)
Comment (bryangaledxw) [reply cypress_shared/fixtures/applicationData.json]: Yep, another thing to revisit when we get to those pages

File: cypress_shared/helpers/apply.ts (hunk 1 above)
Comment (bryangaledxw) [reply cypress_shared/helpers/apply.ts]: Thanks!

**Pull request merged**